django/utils/http.py

```diff
@@ -293,12 +295,21 @@ def is_safe_url(url, host=None):
             url = force_text(url)
         except UnicodeDecodeError:
             return False
+    if allowed_hosts is None:
+        allowed_hosts = set()
+    if host:
+        warnings.warn(
```

Comment:
I agree with @charettes. As we're deprecating the old name we have a new opportunity to use a more explicit and obvious name. It is brand new so doesn't need to match the old name. I think `allowed_hosts` fills that role best. Additionally, I like the symmetry with the setting `ALLOWED_HOSTS`.